### DISCUSSION TITLE
Por algun motivo me retiraron del equipo del launch donde ponen los live

### DISCUSSION THREAD

**Discussion by OATS74** [Launch-X-Latam/MisionFrontEnd#89] [Q&A]
podrian agregarme de nuevo por favor  , apenas llevo dos live y no sabia ni como chatear durante el live .

**Comment by DonQuike** [comment]
De casualidad iniciabas reuniones o hacías cosas que el staff indicaba que no hicieramos? Porque ayer hubo bastantes problemas con cosas así

**Reply by OATS74** [reply]
No.  te juro que no hice nada. Que injusto vale

**Reply by JosafatJimenezB** [reply]
manda mensaje por instagram hablando sobre tu caso para que te den una solución
